stl/inc/random

```diff
@@ -1011,7 +1012,7 @@ struct _Swc_traits { // traits for subtract_with_carry generator
     template <class _Elem, class _Traits>
     static void _Write(

```

Comment:
And here.

Reply:
This is writing to a `basic_ostream` which is not a compile-time thing.